engine/markdown/Options/Plugins/Emitters.md

```diff
@@ -8,8 +8,8 @@
 | `life.delay`    | `number`    | `0.1`                                                                                                                                                              | The emitter delay between any appearance                                                       |
 | `particles`     | `object`    |                                                                                                                                                                    | See Particles options {@link IParticles | here}                                                                    |
 | `position`      | `object`    | `{ "x": 50, "y": 50 }`                                                                                                                                             | The position specified is not absolute, it will be used as a percent value of the canvas size. |
-| `size.width`    | `number`    | `10`                                       | |
-| `size.height`   | `number`    | `10`                                       | |
+| `size.width`    | `number`    | `10`                                       | Width of the generated particles in the unit specified in `size.mode` |
```

Comment:
This size is related to the emitter, if greater than 0 it will be a 2D shape, starting from a point (0,0), a line (x,0) or (y,0) or a square or a circle if both have values and a shape is selected